phpstan.neon

```diff
@@ -15,6 +15,9 @@ parameters:
 		- tests/*/data-php-*/*
 		- tests/Rules/Doctrine/ORM/entity-manager.php
 
+	propertyAlwaysWrittenTags:
+		- Version
```

Comment:
This needs to be implemented differently so that it also works for attributes https://www.doctrine-project.org/projects/doctrine-orm/en/2.11/reference/attributes-reference.html#attrref_version

Reply:
I could add a condition to `PropertiesExtension::isAlwaysWritten()` that would check if the property matches `ClassMetadataInfo::$versionField`. Would that work?